File: app/src/main/java/mariavv/fitnesspal/presentation/handbook/ItemDecoration.java
package mariavv.fitnesspal.presentation.handbook;

import android.graphics.Rect;
import android.support.v7.widget.RecyclerView;
import android.view.View;

class ItemDecoration extends RecyclerView.ItemDecoration {
    private int offset;

    public ItemDecoration(int offset) {
        this.offset = offset;
    }

    @Override
    public void getItemOffsets(Rect outRect, View view,
                               RecyclerView parent, RecyclerView.State state) {

        if (parent.getChildAdapterPosition(view) == 0) {

            outRect.right = offset;
            outRect.left = offset;
            outRect.top = offset;
            outRect.bottom = offset;
        }
    }
}

File: app/src/main/java/mariavv/fitnesspal/presentation/handbook/ItemDecoration.kt
package mariavv.fitnesspal.presentation.handbook

import android.graphics.Rect
import android.support.v7.widget.RecyclerView
import android.view.View

internal class ItemDecoration(private val offset: Int) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View,
                                parent: RecyclerView, state: RecyclerView.State) {

        if (parent.getChildAdapterPosition(view) == 0) {

            outRect.right = offset
            outRect.left = offset
            outRect.top = offset
            outRect.bottom = offset
        }
    }
}
